code/modules/transport/tram/tram_signals.dm

```diff
@@ -659,7 +659,7 @@
 
 	for(var/obj/machinery/transport/guideway_sensor/sensor in SStransport.sensors)
 		if(sensor.z == src.z)
-			if((sensor.x == src.x && sensor.dir & NORTH|SOUTH) || (sensor.y == src.y && sensor.dir & EAST|WEST))
+			if((sensor.x == src.x && sensor.dir & (NORTH|SOUTH)) || (sensor.y == src.y && sensor.dir & (EAST|WEST)))
```

Comment:
```suggestion
			if((sensor.x == src.x && (sensor.dir & (NORTH|SOUTH))) || (sensor.y == src.y && (sensor.dir & (EAST|WEST))))
```

Reply:
Refactored this into early continue statements via https://github.com/tgstation/tgstation/pull/92609/commits/af6e4761681074822471f18748f1d85d39984edb